src/modules/emote_menu/stores/emote-menu-store.js

```diff
@@ -33,13 +34,38 @@ function computeScore({totalUses, recentUses}) {
   return Math.floor((totalUses * frecency) / recentUses.length);
 }
 
+const TWITCH_EMOTE_USAGE_HISTORY_KEY = 'twilight.emote_picker_history';
+
+function loadDefaultFrequentEmotes() {
+  if (getPlatform() !== PlatformTypes.TWITCH || !storage._localStorageSupport) {
+    return DEFAULT_FREQUENT_EMOTES;
+  }
+
+  const history = storage.get(TWITCH_EMOTE_USAGE_HISTORY_KEY, null);
+  if (history == null) {
+    return DEFAULT_FREQUENT_EMOTES;
+  }
+
+  const emoteHistory = {};
+  for (const [emoteId, {uses, lastUpdatedAt}] of Object.entries(history)) {
```

Comment:
if the format of this storage value changes, it could break the emote menu store load. should this be in a try/catch?